File: perl/src/tests/t_read_fetch.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$area = Atrshmlog::get_area();

@r = Atrshmlog::read_fetch($area);

print "read fetch : $r[0] : \n";

if ($r[0] == 0) {
    print "read fetch : $r[1] : $r[3] : \n";
}
 
print " \n";

exit (0);

# end of main

File: python/src/tests/t_read_fetch.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

area = atrshmlog.get_area()

r = atrshmlog.read_fetch(area)

print('read fetch : ' + str(r[0]) + ' : ')

if r[0] == 0 :
    print('read fetch : ' + str(r[1]) + ' : ' + str(r[3]) + ' : ')
    

print (' ')

exit(0)

# end of test

